release/tools/build.xml

```diff
@@ -92,7 +92,7 @@
         <echo message=" jsf           [overrides]"/>
         <echo message=" jstl          [overrides]"/>
         <echo message=" el            [overrides]"/>
-        <echo message=" j2ee          [overrides]"/>
+        <echo message=" jakartae      [overrides]"/>
```

Comment:
Should this echo jakartaee or jakartaee?

Reply:
Corrected this. Thanks